src/ar_http_iface_server.erl

```diff
@@ -638,6 +665,19 @@ handle(Method, [<<"height">>], _Req) when (Method == 'GET') or (Method == 'HEAD'
 handle(_, _, _) ->
 	{400, [], <<"Request type not found.">>}.
 
+handle_get_tx(Hash) ->
+	case hash_to_maybe_filename(tx, Hash) of
+		{error, ID, unavailable} ->
+			case is_a_pending_tx(ID) of
+				true ->
+					{error, pending};
+				false ->
+					ar_tx_db:get(ID)
```

Comment:
`ar_tx_db:get/1` can return `not_found` but it looks like this function expect it to return `{error, not_found}`.